liveblog/liveblog
# [LBSD-2815] - Instagram images are not showing up all the time

### Purpose

This PR fixes an issue where the Instagram embeds are not getting loaded properly due to the object `instgrm` not being defined from the embed script.

### What has changed

- Changed the `src` for the embed script to the new link to fetch the script directly and without redirection.

### Steps to test

1. Log in to the Editor. Add an Instagram embed post. 
2. Click Live to see the live preview. Observe that the Instagram post loads correctly.

Resolves: [LBSD-2815](https://sofab.atlassian.net/browse/LBSD-2815)
